### ISSUE TITLE
Chinese training data format can only be JSON

### ISSUE THREAD

**Issue by babyeleven** [crownpku/Rasa_NLU_Chi#86]
Chinese training data format can only be json, I use MD format training error, prompt:

C: 3.13469   f-score: 1
C: 1.57234   f-score: 1
C: 0.791172   f-score: 1
C: 0.400586   f-score: 1
best C: 0.791172
test on train:
15

overall accuracy: 1
Part II: elapsed time: 0 seconds.
df.number_of_classes(): 1
Fitting 2 folds for each of 6 candidates, totalling 12 fits
C:\Users\37284\AppData\Local\Programs\Python\Python36\lib\site-packages\sklearn\metrics\classification.py:1135: UndefinedMetricWarning: F-score is ill-defined and being set to 0.0 in labels with no predicted samples.
  'precision', 'predicted', average, warn_for)
C:\Users\37284\AppData\Local\Programs\Python\Python36\lib\site-packages\sklearn\metrics\classification.py:1135: UndefinedMetricWarning: F-score is ill-defined and being set to 0.0 in labels with no predicted samples.
  'precision', 'predicted', average, warn_for)
C:\Users\37284\AppData\Local\Programs\Python\Python36\lib\site-packages\sklearn\metrics\classification.py:1135: UndefinedMetricWarning: F-score is ill-defined and being set to 0.0 in labels with no predicted samples.
  'precision', 'predicted', average, warn_for)
[Parallel(n_jobs=1)]: Done  12 out of  12 | elapsed:    0.0s finished
